src/App.php

```diff
@@ -2,33 +2,50 @@
 
 namespace TrendingRepos;
 
-class App {
-
-    /**
-     * @var array
-     */
-    protected static $registry = [];
+use TrendingRepos\Exception\RouterException;
+use TrendingRepos\Core\Request;
+use TrendingRepos\Core\Router;
+
+class App 
+{
+    public $registry = [];
```

Comment:
What if I'll tell you that this property shouldn't be exposed as well? 🔒

Reply:
How ? controllers need to reach this property 
i want to make something like this in controllers 
`$this->app->registry['config']['offset']`
how to do this if it's not exposed